packages/common/src/components/Preview/index.tsx

```diff
@@ -118,6 +117,15 @@ class BasePreview extends React.Component<Props, State> {
     }
   }
 
+  currentUrl = () => {
```

Comment:
@CompuIves You can  just use an implicit return here

```js
currentUrl = () => (
  this.props.url ||
  (this.serverPreview
    ? getSSEUrl(this.props.sandbox, this.props.initialPath)
    : frameUrl(this.props.sandbox, this.props.initialPath || ''))
  );
```

Reply:
Hmm good one, the linter should've caught this.